pkg/universe.dagger.io/alpha/go/golangci/lint.cue

```diff
@@ -8,8 +8,14 @@ import (
 	"universe.dagger.io/go"
 )
 
+// Like #LintBase, but with a pre-configured container image.
+#Lint: #LintBase & {
+	_image: #Image
+	input:  _image.output
+}
+
 // Lint using golangci-lint
-#Lint: {
+#LintBase: {
```

Comment:
Some alternatives I came up with in the discussion:

- Template (eg. `LintTemplate`)
- Command (eg. `LintCommand`)
- Definition (or Def) (eg. `LintDefinition`) (this is somewhat redundant)
- Proto (eg. `LintProto`)

For some reason, Template and Definition are the closest to me, but I can't reason about them. WDYT?

Reply:
I personally like it